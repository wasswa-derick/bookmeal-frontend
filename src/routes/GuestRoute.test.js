import React from "react";
import { shallow } from "enzyme";
import { GuestRoute } from "./GuestRoute";

describe("Guest Route", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <GuestRoute isUserAuthenticated={false} component={HTMLDivElement} />
    );
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
