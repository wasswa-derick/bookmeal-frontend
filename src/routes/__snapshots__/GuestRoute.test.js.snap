// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Guest Route should render correctly 1`] = `
<Route
  render={[Function]}
/>
`;
